spiffe-helper.go

```diff
@@ -15,19 +15,18 @@ import (
 	"os/signal"
 	"path"
 	"strings"
-	"sync"
 	"syscall"
 	"time"
+	"go.uber.org/atomic"
```

Comment:
nit: i'm not sure its worth pulling in an extra dependency for one atomic boolean that can be modeled with as an int with a 0 (false) or non-zero value (true).

Reply:
That could be modeled with an int, and no need to use  "atomic/sync" package ?

Reply:
you'd still use atomic/sync to load the value and to swap it with 0 or 1 when the process state changed.

Reply:
Refactored that flag using atomic/sync int